src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/DependencyGraphSpecRequestProvider.cs

```diff
@@ -155,7 +157,8 @@ private static ExternalProjectReference GetExternalProject(PackageSpec rootProje
                 RestoreOutputPath = project.PackageSpec.RestoreMetadata.ProjectStyle == ProjectStyle.ProjectJson ? rootPath : project.PackageSpec.RestoreMetadata.OutputPath,
                 DependencyGraphSpec = projectDgSpec,
                 MSBuildProjectExtensionsPath = projectPackageSpec.RestoreMetadata.OutputPath,
-                ParentId = restoreArgs.ParentId
+                ParentId = restoreArgs.ParentId,
+                SignedPackageVerifierSettings = SignedPackageVerifierSettings.GetDefault(revocationMode: revocationMode)
```

Comment:
Should you pass the whole verification settings or just revocation mode? Do you foresee more verification settings me configurable? if not, then I'll recommend to pass the specific parameter.

Reply:
Yeah, soon we will be reading also information about client policies and trusted signers, we need a solution that is simple enough to later add more things that are read from settings.